beetsplug/unimported.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
```

Comment:
Can you please copy the standard comment header from another Python file to this one?